files/image_config/caclmgrd/caclmgrd

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/env python
+#
+# caclmgrd
+#
+# Control plane ACL manager daemon for SONiC
+#
+#  Upon starting, this daemon reads control plane ACL tables and rules from
+#  Config DB, converts the rules into iptables rules and installs the iptables
+#  rules. The daemon then indefintely listens for notifications from Config DB
+#  and updates iptables rules if control plane ACL configuration has changed.
+#
+
+try:
+    import os
+    import subprocess
+    import sys
+    import syslog
+    from swsssdk import ConfigDBConnector
+except ImportError as err:
+    raise ImportError("%s - required module not found" % str(err))
```

Comment:
Why do we need this?

Reply:
This ImportError try/except block is something that was present in many SONiC Python scripts when I first joined the team. It is something I have carried over into new scripts for consistency. It is not necessary, it just presents a clear error message to the user if it fails to import a module and attempts to continue on.

It might be better to simply throw the exception and exit; again, I've simply carried on using it for consistency within the project.